sycl/include/CL/sycl/ONEAPI/atomic_enums.hpp

```diff
@@ -23,12 +23,13 @@ namespace sycl {
 namespace ONEAPI {
 
 enum class memory_order : int {
-  relaxed,
-  acquire,
-  __consume_unsupported, // helps optimizer when mapping to std::memory_order
-  release,
-  acq_rel,
-  seq_cst
+  relaxed = 0,
```

Comment:
Do these match their std:: versions?

Reply:
According to https://en.cppreference.com/w/cpp/atomic/memory_order it does not. However, we can't do anything at the moment. Changing order of these enums would be a breaking change.